tests/test_settings.py

```diff
@@ -16,6 +16,7 @@ def test_collect_settings(imscc_file):
         "input_files":  {imscc_file},
         "output_format": parsed_args.result,
         "workspace": Path.cwd() / "output",
+        'link_file': None,
```

Comment:
Nit: other keys in this dict in double quotes.

I tried to persuade people include [black](https://github.com/psf/black) in CI pipeline, but didn't succeed :grinning: